lib/common/template.js

```diff
@@ -108,5 +108,18 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
         });
     };
 
+    TemplateService.prototype.unlink = function (name, scope) {
```

Comment:
unlink is consistent with the node `fs` module's naming convention, so perhaps that's why?